client/core/core.go

```diff
@@ -648,8 +655,14 @@ func (c *Core) OpenWallet(assetID uint32, appPW string) error {
 	if err != nil {
 		return err
 	}
-	dcrID, _ := dex.BipSymbolID("dcr")
-	if assetID == dcrID {
+
+	state := wallet.state()
+	avail, locked, _ := wallet.Balance(0) // the wallet just unlocked successfully
+	log.Infof("Connected to and unlocked %s wallet. Account %q balance available "+
+		"= %d / locked = %d, Deposit address = %s",
+		state.Symbol, wallet.Account, avail, locked, state.Address)
```

Comment:
A common debugging step so far has been "run `getbalances` and `listlockunspent`", and I think this will help with that.